src/OpenTelemetry.Api/Logs/LogRecordSeverityExtensions.cs

```diff
@@ -0,0 +1,83 @@
+// <copyright file="LogRecordSeverityExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#nullable enable
+
+namespace OpenTelemetry.Logs;
+
+/// <summary>
+/// Contains extension methods for the <see cref="LogRecordSeverity"/> enum.
+/// </summary>
+internal static class LogRecordSeverityExtensions
+{
+    private static readonly string[] LogRecordSeverityShortNames = new string[]
```

Comment:
Get this static initialization invoked in the `OpenTelemetryLoggerProvider` ctor?

Reply:
I'm not opposed to it, but I think most applications will never have a need to get the short name / display name so it might be better to leave it lazy?